angular-workspace/components/annotation/annotation.angular-template.ts

```diff
@@ -0,0 +1,28 @@
+import { StoryFnAngularReturnType } from "@storybook/angular/dist/ts3.9/client/preview/types";
+import { Annotation } from "dso-toolkit";
+
+import { ComponentImplementation } from "../../templates";
+
+export const angularAnnotation: ComponentImplementation<Annotation<StoryFnAngularReturnType>> = {
+  component: "annotation",
+  implementation: "angular",
+  template: () =>
+    function annotation(props) {
+      return {
+        props,
+        template: `
+          <dso-annotation-button [identifier]="annotationButton.identifier"></dso-annotation-button>
+          <dso-annotation-output
+            [slot]="annotationOutput.slotName"
+            [attr.identifier]="annotationOutput.identifier"
+            [annotationPrefix]="annotationOutput.prefix"
+            (dsoToggleAnnotation)="annotationOutput.dsoToggleAnnotation?.($event)"
+          >
+            ${props.annotationOutput.title.template}
```

Comment:
Ik denk dat ik nu deels snap wat er mis gaat met de child-templates. Een `angular-template.ts` krijgt props op het component gezet. In deze file interpoleren we niet, maar is bv annotationOutput.prefix beschikbaar in het angular component omdat `annotationOutput` als `Input()` is gemaakt door iets slims.

Als je vervolgens van een angular-template alleen de `.template` (en dus niet de props) interpoleert ontbreken de benodigde `Input()`'s op het component.